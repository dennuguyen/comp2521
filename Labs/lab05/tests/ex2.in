C
r 5 6
C
p 6 9
b 5 9
q
