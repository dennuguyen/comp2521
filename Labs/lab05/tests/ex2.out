Graph has V=10 and E=16
V    Connected to
--   ------------
0    1 2 5
1    0 5
2    0 3
3    2 4 5 8
4    3 5 7 8
5    0 1 3 4 6 7
6    5
7    4 5 8 9
8    3 4 7 9
9    7 8

> Component 0 has 0 1 2 3 4 5 6 7 8 9

Graph has V=10 and E=16
V    Connected to
--   ------------
0    1 2 5
1    0 5
2    0 3
3    2 4 5 8
4    3 5 7 8
5    0 1 3 4 6 7
6    5
7    4 5 8 9
8    3 4 7 9
9    7 8

> 
Graph has V=10 and E=15
V    Connected to
--   ------------
0    1 2 5
1    0 5
2    0 3
3    2 4 5 8
4    3 5 7 8
5    0 1 3 4 7
6   
7    4 5 8 9
8    3 4 7 9
9    7 8

> Component 0 has 0 1 2 3 4 5 7 8 9
Component 1 has 6

Graph has V=10 and E=15
V    Connected to
--   ------------
0    1 2 5
1    0 5
2    0 3
3    2 4 5 8
4    3 5 7 8
5    0 1 3 4 7
6   
7    4 5 8 9
8    3 4 7 9
9    7 8

> No path exists from 6 to 9

Graph has V=10 and E=15
V    Connected to
--   ------------
0    1 2 5
1    0 5
2    0 3
3    2 4 5 8
4    3 5 7 8
5    0 1 3 4 7
6   
7    4 5 8 9
8    3 4 7 9
9    7 8

> x=5   q=0>1>3>4>7
x=0   q=1>3>4>7>1>2
x=1   q=3>4>7>1>2
x=3   q=4>7>1>2>2>4>8
x=4   q=7>1>2>2>4>8>7>8
x=7   q=1>2>2>4>8>7>8>8>9
x=2   q=2>4>8>7>8>8>9
x=8   q=7>8>8>9>9
x=9   q=9

Graph has V=10 and E=15
V    Connected to
--   ------------
0    1 2 5
1    0 5
2    0 3
3    2 4 5 8
4    3 5 7 8
5    0 1 3 4 7
6   
7    4 5 8 9
8    3 4 7 9
9    7 8

> 